packages/workbox-core/test/bundle/node/utils/test-logHelper.js

```diff
@@ -0,0 +1,339 @@
+import chai from 'chai';
+import sinon from 'sinon';
+
+import logHelper from '../../../../src/utils/logHelper';
+
+const expect = chai.expect;
+
+describe(`logHelper [${process.env.NODE_ENV}]`, function() {
+  let stubs = [];
+
+  beforeEach(function() {
+    logHelper.logLevel = logHelper.LOG_LEVELS.verbose;
+  });
+
+  afterEach(function() {
```

Comment:
I'd love to see more consistency.

Speaking for myself, I've learned a lot about testing best practices while working on v1, and I know that my newer tests have evolved quite a bit from what I was writing initially.

I worry about veering too far down the path of writing our own wrappers on top of `mocha`/`chai`/`sinon`, and then we've ended up effectively writing our own testing framework, but... that wouldn't be the worst idea?

I don't know about linting-style rules for test consistency at this point, because we don't need linting to tell us that the tests are inconsistent. We could impose some rigor via linting  once we have a baseline of consistency.